unit-tests/func/frames/test_first_frame_delay.py

```diff
@@ -0,0 +1,68 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+# test:device L500*
+# test:device D400*
+
+import pyrealsense2 as rs
+from rspy.stopwatch import Stopwatch
+from rspy import test
+import time
+import platform
+
+# Start depth + color streams and measure the time from stream opened until first frame arrived.
+# Verify that the time do not exceeds the maximum time allowed
+MAX_TIME_TO_WAIT_FOR_FIRST_FRAME = 2.5  # [sec]
```

Comment:
Add that this is the KPI used in validation?

Reply:
They use it for pipeline.. but I can add it if you want.

Reply:
It would be nice if this test described the timing constraints as we've learned them: that FW requires up to 1-1.5 sec to...

Reply:
Added a comment